### PR TITLE
Bdh cleanup

### PR DESCRIPTION
**Pull request by bryan-howard**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Removes trailing whitespace.
Addresses all outstanding shellcheck warnings.

Does this close any currently open issues?
------------------------------------------
I don't think so.

Any relevant logs, error output, etc?
-------------------------------------
No.

Any other comments?
-------------------
No.

### CODE REVIEW

File: src/main.sh [hunk 1]
```diff
@@ -41,7 +34,9 @@ function _show_version {
 
 function _init_script {
   if [[ $# == 0 ]]; then
-    _incorrect_usage 'no input parameters provided.' 126
```
Comment (sobolevn) [inline src/main.sh:44]: Why did you change this?

**Comment by joshrabinowitz** [comment]
@sobolevn @bryan-howard I'm planning to merge this soon, then revert the `_incorrect_usage` call commented on above.

File: src/main.sh (hunk 1 above)
Comment (bryan-howard) [reply src/main.sh:44]: shellcheck correctly warns that the usage() function expects parameters but is never called with any. I have just submitted a different pull request based on the current master which passes the rest of the parameter list to the usage function rather than inline it. So you can forget this part of this submission.
